dapp/src/components/page/home/MyCredentials.svelte

```diff
@@ -18,25 +13,22 @@
 
   onMount(async () => {
     try {
-      if (Object.values($claimsStream).every((claim) => !claim.url)) {
+      if (
+        Object.values($claimsStream).every(
+          (claim) => !claim.content && !claim.preparedContent
+        )
+      ) {
         data = [];
       } else {
         data = await Promise.all(
           Object.values($claimsStream)
-            .filter((claim) => claim.url)
+            // TODO: Distinguish between content and preparedContent
```

Comment:
What's prepared content?

Reply:
It's a complete claim that hasn't been saved to Kepler or added the the storage. The UI doesn't distinguish between saved claims and unsaved claims currently, but if we want to add an edit flow, we'll want to know the difference and divide them accordingly.